### PR TITLE
Add translation contributor document

### LINKED ISSUE #305: Document the process to contribute to translation and localization
Hi,

AFAIK there is no reference to the translation process neither in the git repository or on https://wiki.gnome.org/Apps/GTG

Someone planning to help translate gtg should find a brief Howto somewhere, or at least a reference of https://translations.launchpad.net/gtg

Cheers

### PR DESCRIPTION
**Pull request by Neui**
Basically my current workflow. Feedback welcome!
Assumes that the translator deals with the files directly and not use like an web interface to translate or something.

fixes #305

### CODE REVIEW

**Comment by diegogangl** [comment]
This looks good. We eventually want to move to Gnome's gitlab and use Damned Lies for translations, but it's not clear when/if that's going to be happen.

@leio it'd be great if you could take a quick look to this since you know more about translations

File: docs/contributors/translating.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+You may want to reset the not modified translation files to prevent git showing you a lot of uncommited files that could interfere when switching branches.
+Make sure to this AFTER COMMITING YOUR CHANGES!
+
+    git checkout -- po # drop uncommited translation changes
+
+# Potential errors
+
+* Not removing the `#, fuzzy` line/part, which causes the translation not to apply
+* During launching, it might complain about certain files not being found in [`po/POTFILES.in`][POTFILES.IN].
+  It is save to remove the lines from that file and re-run until it works.
```
Comment (gerazo) [inline docs/contributors/translating.md:74]: typo: "It is safe"

**Comment by gerazo** [comment]
Nice doc. I wish I had something like this when starting...

File: docs/contributors/translating.md (hunk 1 above)
Comment (Neui) [reply docs/contributors/translating.md]: Thank you!

**Comment by diegogangl** [comment]
Thanks!

**Pull request merged**